### PR TITLE
add broker bot

### PR DESCRIPTION
**Pull request by dimitri-da**
This PR add a bot for the broker. The functionality at the moment is limited to accepting broker order requests and assigning an id to them

### CODE REVIEW

**Review by Sarahb-da — APPROVED** [review]
looks good

File: automation/broker/bot/broker_bot.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    network = dazl.Network()
+    network.set_config(url=url)
+
+    logging.info(f'Bot will run under party: {broker_party}')
+
+    client = network.aio_party(broker_party)
+
+    @client.ledger_ready()
+    def say_hellpo(event):
```
Comment (alexmatson-da) [inline automation/broker/bot/broker_bot.py:36]: I think you may have meant `say_hello` here

File: automation/broker/bot/broker_bot.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        logging.info("DA Marketplace Broker bot is ready!")
+
+    @client.ledger_created(MARKETPLACE.BrokerOrderRequest)
+    def handle_deposit_transfer_request(event):
```
Comment (alexmatson-da) [inline automation/broker/bot/broker_bot.py:40]: Maybe `handle_broker_order_request`?

**Review by alexmatson-da — APPROVED** [review]
Looks good! Just some minor suggestions for naming the handlers

**Review by alexgraham-da — APPROVED** [review]

**Pull request merged**